analyzer/src/test/kotlin/managers/utils/PubCacheReaderTest.kt

```diff
@@ -81,5 +87,41 @@ class PubCacheReaderTest : WordSpec({
                 )
             ) shouldBe hostedPackageCacheDir
         }
+
+        "resolve the relative path of a local dependency" {
+            PubCacheReader().findProjectRoot(
+                jsonMapper.readTree(
+                    """
+                        {
+                            "dependency": "transitive",
+                            "description": {
+                                "path": "$RELATIVE_PATH",
+                                "relative": true
+                            },
+                            "source": "path"
+                        }
+                    """.trimIndent()
+                ),
+                File(Os.env["PUB_CACHE"])
```

Comment:
You also need to add this line to the existing test cases above now.

Reply:
done